### PR TITLE
add : login-signup

### PR DESCRIPTION
**Pull request by Varun-28**
Add login and signup form to my ecomm site.

login
signup

### CODE REVIEW

File: css/component_styles/signin.css [hunk 1]
```diff
@@ -7,4 +7,58 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    box-shadow: var(--card-shadow);
+    min-height: 30rem;
+    min-width: 30rem;
+}
+
+.login-signup-form>* {
```
Comment (AnuragSahu11) [inline css/component_styles/signin.css]: could add a space before >* to make it more visible

**Review by AnuragSahu11 — APPROVED** [review]
Looks good , the Ui also looks clean, you can merge

File: components/signup.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Shoebox</title>
+    <link rel="icon" type="image/x-icon" href="../assets/icon.png">
+
+    <!-- stylesheets -->
+    <link rel="stylesheet" href="../css/component_styles/signup.css">
+
+    <!-- font awesome -->
+    <script src="https://kit.fontawesome.com/e413b10342.js" crossorigin="anonymous"></script>
```
Comment (Hamdrive) [inline components/signup.html:15]: Consider importing this directly in your main root CSS file, allows you to automatically have it imported into your CSS components and HTML

**Review by Hamdrive — COMMENTED** [review]
Great code written, excellent use of comments and overall code formatting
Have made some optional suggestions, else code is ready to merge!

File: components/signup.html (hunk 2 above)
Comment (Varun-28) [reply components/signup.html:15]: okh I will do it, 👍 thanks for the rest

File: css/component_styles/signin.css (hunk 1 above)
Comment (Varun-28) [reply css/component_styles/signin.css]: ok i will do it

**Pull request merged**